src/projectscene/qml/Audacity/ProjectScene/trackspanel/TrackEffectList.qml

```diff
@@ -0,0 +1,45 @@
+/*
+ * Audacity: A Digital Audio Editor
+ */
+import QtQuick
+import QtQuick.Layouts
+import QtQuick.Controls
+
+import Muse.Ui
+import Muse.UiComponents
+
+import Audacity.ProjectScene
+
+Rectangle {
+    id: root
+
+    property alias preferredHeight: trackEffectList.height
+
+    visible: trackEffectList.count > 0
+
+    RtEffectListModel {
+        id: trackEffectListModel
+        trackId: view.itemAtIndex(effectsPanel.selectedTrackIndex).item.trackId
+        Component.onCompleted: {
```

Comment:
better to use Component.onCompleted for root item

Reply:
Why ?